### ISSUE TITLE
[babylon] Handle the early unbonding transaction

### ISSUE THREAD

**Issue by jolestar** [rooch-network/rooch#2778]
### Proposal Details

The staking UTXO can be spent before the time lock via early unbonding, we need to handle it.
When the UTXO is spent, we should remove the `BBNStakeSeal` object.

https://github.com/babylonlabs-io/staking-indexer/issues/27

**Comment by mx819812523** [comment]
can we temporarily use`BBNStakeSeal.txid` and `BBNStakeSeal.staking_output_index` call `object::exists_object_with_type<UTXO>` to check BBNStakeSeal is effective?

**Comment by jolestar** [comment]
Yes, we can provide a function to check UTXO and remove the BBNStakeSeal. After the Babylon mainnet is ready, we design an automated processing mechanism to clear the expired BBNStakeSeal.
